tox.ini

```diff
@@ -11,8 +11,15 @@ envlist =
 skip_missing_interpreters = true
 
 [testenv]
-deps = pytest
+passenv = CIRCLECI CIRCLE_* CI_PULL_REQUEST COVERALLS_PARALLEL COVERALLS_REPO_TOKEN COVERALLS_SERVICE_NAME
+
+deps =
+    pytest
+    coveralls
 whitelist_externals = make
 commands =
     make test_all
     pip3 install geekhours
+    coverage run --source=geekhours setup.py test
+    coverage report -m --omit=.tox/*
+    coveralls --verbose
```

Comment:
Let's omit verbose log. We confirmed coveralls works well.

Reply:
Fixed.